### PR TITLE
Fix LOCALISE_TIMESTAMP=False and document better how LOCALISE_TIMESTAMP works

### PR DESCRIPTION
**Pull request by gdubicki**
Fixes https://github.com/voxpupuli/puppetboard/issues/461

Plus various fixes for running tests.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,7 +32,6 @@ coverage.xml
 .tox
 .mypy_cache
 .pytest_cache
-nosetests.xml
```
Comment (bastelfreak) [inline .gitignore:35]: why did you remove it from .gitignore? The file isn't part of this commit.

Comment (gdubicki) [reply .gitignore:35]: Because we are not using nosetests in this project.

Comment (gdubicki) [reply .gitignore:35]: I noticed it while I was fixing flake8 issues.

Comment (gdubicki) [reply .gitignore:35]: Is that ok, @bastelfreak?

**Review by bastelfreak — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (gdubicki) [reply .gitignore:35]: Thank for the approval, @bastelfreak !
